subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -743,6 +793,32 @@ static void after_cen_offset_get(uint16_t conn_interval, uint32_t ticks_slot,
 }
 #endif /* CONFIG_BT_CONN */
 
+#if defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_BROADCASTER)
+static bool ticker_adv_aux_match_op_cb(uint8_t ticker_id, uint32_t ticks_slot,
+				       uint32_t ticks_to_expire,
+				       void *op_context)
+{
+	ARG_UNUSED(ticks_slot);
+	ARG_UNUSED(ticks_to_expire);
+	ARG_UNUSED(op_context);
+
+	return (ticker_id >= TICKER_ID_ADV_AUX_BASE) &&
+	       (ticker_id <= TICKER_ID_ADV_AUX_LAST);
```

Comment:
Consider `IN_RANGE`:
```suggestion
	return IN_RANGE(ticker_id, TICKER_ID_ADV_AUX_BASE, TICKER_ID_ADV_AUX_LAST);
```